src/containers/Routing/Routing.jsx

```diff
@@ -0,0 +1,68 @@
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Home from "../Home/Home";
+import Profile from "../Profile/Profile";
+import userProfile from "../../assets/data/dummyUserData";
+import PathwaysMenu from "../PathwaysMenu/PathwaysMenu";
+import Marketplace from "../Marketplace/Marketplace";
+import CourseOverview from "../CourseOverview/CourseOverview";
+import ArticleIndex from "../ArticleIndex/ArticleIndex";
+import React from "react";
+import LessonOverview from "../LessonOverview/LessonOverview";
+import MarketplaceIndex from "../MarketplaceIndex/MarketplaceIndex";
+import PathwayOverview from "../PathwayOverview/PathwayOverview";
+import SignIn from "../SignIn/SignIn";
+import Splash from "../Splash/Splash";
+import CreateAccount from "../CreateAccount/CreateAccount";
+import Articles from "../Articles/Articles";
+import SkillsTree from "../SkillsTree/SkillsTree";
+import Quiz from "../Quiz/Quiz";
+
+const Routing = () => {
+  return (
+    <Router>
+      <Routes>
+        <Route
+          path="/marketplace"
+          element={<Marketplace userProfile={userProfile} />}
+        />
+        <Route
+          exact
+          path="/marketplace/:marketplaceId"
+          element={<MarketplaceIndex />}
+        />
+
+        <Route path="/pathways" element={<PathwaysMenu />} />
+        <Route
+          exact
```

Comment:
You should need the `exact` prop anymore this is a hangover from the previous version of react-router. I may be wrong though, do you have to use it?

Reply:
I've taken the exacts out, and it still works. Must've copied that from somewhere